### PR TITLE
New channel reporting to the admin channel

### LINKED ISSUE #15: Add new channel reporting to the admin channel
## Description
Reinstate new channel reporting to the admin channel.

With this feature, the following should happen:

1. A new channel is created
2. The application receives an event with the information on the channel
3. The application sends a message through automod to the admin channel to give details about the event

## Acceptance Criteria
- [ ]  Application can receive events for when an emoji is added, removed, or when a new alias is created
- [ ] Bot posts a message about the event in the admin channel

## Extra
[Here](https://github.com/jpmcb/OSU-Auto-Mod/) is a link to the old automod repo for how this used to be implemented. [Here](https://api.slack.com/events/channel_created) is the event api we will most likely use.

### PR DESCRIPTION
**Pull request by hschallhorn**
Resolves #15 

## Description
Listens to new channel events and reports them to the admin channel

## Acceptance  criteria
- [ ] Automod is subscribed to new_channel events
- [ ] Automod sends a message to the admin channel any time a new channel is created

## Testing
1. Pull this branch
1. Start the application
```bash
SLACK_BOT_TOKEN=<integration_testing_api_key>
export SLACK_BOT_TOKEN
python app.py
```
1. Create a new channel
1. Ensure automod sends a message about the new channel to the admin channel

### CODE REVIEW

File: emoji_message.py [hunk 1]
```diff
@@ -4,17 +4,17 @@
...
         report on.
 
-        :param channel: The channel to send the emoji message to
+        :param report_channel: The channel to send the emoji message to
         :param icon_emoji: The name of the emoji
         :param event_type: The event that has happened, 'add' or 'remove'
         """
-        self.channel = channel
+        self.report_channel = report_channel
         self.username = 'automod'
```
Comment (sarahmaas) [inline emoji_message.py:20]: We should probably be moving stuff like this to a constant file somewhere....eventually.

**Comment by hschallh** [comment]
Merge conflicts fixed!

**Review by sarahmaas — APPROVED** [review]

**Comment by hschallh** [comment]
Going to test and deploy emoji_meta first

**Comment by sarahmaas** [comment]
Emoji meta looks good in integration ❤️

**Pull request merged**